EDR_telem.json

```diff
@@ -652,7 +652,7 @@
     "LimaCharlie":"Pending Response",
     "MDE":"No",
     "Qualys":"No",
-    "Sentinel One":"No",
+    "Sentinel One":"Yes",
```

Comment:
Please change this to "No". The rest looks good.

Reply:
Done